.circleci/install_benchmark_sys_deps.sh

```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/env bash
+
+# to get the add-apt-repository command
+sudo apt-get install software-properties-common
+
+sudo add-apt-repository -y ppa:ethereum/ethereum
+sudo add-apt-repository -y ppa:ethereum/ethereum-dev
+sudo sed -i 's/cosmic/bionic/g' /etc/apt/sources.list.d/ethereum-ubuntu-ethereum-cosmic.list
+sudo sed -i 's/cosmic/bionic/g' /etc/apt/sources.list.d/ethereum-ubuntu-ethereum-dev-cosmic.list
+sudo apt-get -y update
+sudo apt-get install -y --allow-unauthenticated solc
```

Comment:
This is almost certainly going to cause a future CI test failure.  Can we just pre-compile the binary and keep note of the compiler version?  Also, using different compiler versions will end up with different benchmarking results due to different resulting bytecode.

Reply:
True that. Cost me far too long to figure out the right CI setup but I'm not married to that approach. I see your point. See my second comment further down.